platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/reconnect/ReconnectRunner.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.benchmark.reconnect;
+
+import com.swirlds.benchmark.BenchmarkKey;
+import com.swirlds.benchmark.BenchmarkValue;
+import com.swirlds.common.merkle.MerkleInternal;
+import com.swirlds.common.merkle.MerkleNode;
+import com.swirlds.common.merkle.synchronization.config.ReconnectConfig;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.virtualmap.VirtualMap;
+import com.swirlds.virtualmap.internal.pipeline.VirtualRoot;
+
+/**
+ * A runner for reconnect tests.
+ */
+public class ReconnectRunner {
+    /**
+     * Run reconnect with the given teacher and learner maps.
+     *
+     * @param teacherMap a teacher map aka a good, desired state
+     * @param learnerMap a learner map aka a stale, outdated, "wrong" state
+     * @throws Exception thrown if the reconnect code throws any exceptions
+     */
+    public static void reconnect(
```

Comment:
Why isn't this method a part of ReconnectBench? This ReconnectRunner class looks redundant (especially if its `createTreeForMap()` method is eliminated in favor of `MerkleBenchmarkUtils.buildLessSimpleTree()`)